### PR TITLE
fix issue 4274

### PR DESCRIPTION
**Pull request by cangfengzhs**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by jievince — APPROVED** [review]

**Review by SuperYoko — APPROVED** [review]

**Comment by critical27** [comment]
Close #4274

File: src/kvstore/raftex/RaftPart.cpp [hunk 1]
```diff
@@ -1063,7 +1063,6 @@ void RaftPart::processAppendLogResponses(const AppendLogResponses& resps,
       */
       auto [code, lastCommitId, lastCommitTerm] = commitLogs(std::move(walIt), true, true);
       if (code == nebula::cpp2::ErrorCode::SUCCEEDED) {
-        stats::StatsManager::addValue(kCommitLogLatencyUs, execTime_);
```
Comment (critical27) [inline src/kvstore/raftex/RaftPart.cpp:1066]: Could add the `kCommitLogLatencyUs` in the scope from line 1054 to 1082?

File: src/kvstore/Part.cpp [hunk 2]
```diff
@@ -214,7 +215,9 @@ std::tuple<nebula::cpp2::ErrorCode, LogID, TermID> Part::commitLogs(
     std::unique_ptr<LogIterator> iter, bool wait, bool needLock) {
   // We should apply any membership change which happens before start time. Because when we start
   // up, the peers comes from meta, has already contains all previous changes.
-  SCOPED_TIMER(&execTime_);
+  SCOPED_TIMER([](uint64_t elapsedTime) {
+    stats::StatsManager::addValue(kCommitLogLatencyUs, elapsedTime);
+  });
```
Comment (panda-sheep) [inline src/kvstore/Part.cpp:220]: Good

**Review by panda-sheep — APPROVED** [review]

File: src/kvstore/raftex/RaftPart.cpp (hunk 1 above)
Comment (cangfengzhs) [reply src/kvstore/raftex/RaftPart.cpp:1066]: There are two places that will call commitlogs, so if you only add it to the place you said, then the commit latency will record less data. If you add it to both places, then the meaning of this metric will be unclear

Comment (critical27) [reply src/kvstore/raftex/RaftPart.cpp:1066]: How about add it in `commitLogs` itself, that could be unified.

Comment (cangfengzhs) [reply src/kvstore/raftex/RaftPart.cpp:1066]: In this PR, it is modified in this way

Comment (critical27) [reply src/kvstore/raftex/RaftPart.cpp:1066]: 👌 my bad...

**Review by critical27 — APPROVED** [review]

**Pull request merged**